lib/Analysis/Memory/DefinedMemory.cpp

```diff
@@ -24,17 +24,21 @@
 
 #include "tsar/Analysis/Memory/DefinedMemory.h"
 #include "tsar/Analysis/DFRegionInfo.h"
+#include "tsar/Analysis/Memory/Delinearization.h"
 #include "tsar/Analysis/Memory/EstimateMemory.h"
 #include "tsar/Analysis/Memory/MemoryAccessUtils.h"
 #include "tsar/Analysis/Memory/Utils.h"
 #include "tsar/Support/Utils.h"

```

Comment:
Убрал `#include` из `.h` файла.

```suggestion
#include "tsar/Support/GlobalOptions.h"
#include "tsar/Support/Utils.h"
```